pkg/kubelet/rkt/rkt.go

```diff
@@ -464,10 +541,18 @@ func (r *Runtime) makePodManifest(pod *api.Pod, pullSecrets []api.Secret) (*appc
 		}
 		appName := appctypes.MustACName(name)
 
+		kubehash := kubecontainer.HashContainer(&c)
+
 		manifest.Apps = append(manifest.Apps, appcschema.RuntimeApp{
 			Name:  *appName,
 			Image: appcschema.RuntimeImage{ID: *hash},
 			App:   imgManifest.App,
+			Annotations: []appctypes.Annotation{
+				{
+					Name:  *appctypes.MustACIdentifier(k8sRktContainerHashAnno),
+					Value: strconv.FormatUint(kubehash, 10),
+				},
+			},
```

Comment:
@dgonyeo Would you like to take a chance to refactor the loop? maybe move to a function called `makeAppManifest` or something.